### PR TITLE
chore: create bug report issue template

### PR DESCRIPTION
**Pull request by rouille**
Same as https://github.com/Breakthrough-Energy/PowerSimData/pull/454

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+#
+```
+
+### Expected outcome
+A description of the expected outcome from the code snippet.
+
+### Environment
+Please specify your platform and versions of the relevant libraries you are using:
+* Operating system:
+* PowerSimData revision (run `git rev-parse origin/HEAD`):
```
Comment (jon-hagg) [inline .github/ISSUE_TEMPLATE/bug_report.md:41]: Should it be PostREISE?

Comment (rouille) [reply .github/ISSUE_TEMPLATE/bug_report.md:41]: Oops

Comment (rouille) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

**Review by jon-hagg — APPROVED** [review]

**Review by BainanXia — APPROVED** [review]
Looks good.

**Pull request merged**